halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/plugins/Manifest.java

```diff
@@ -22,23 +22,22 @@
 import java.util.*;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import java.util.stream.Stream;
 import lombok.Data;
 import lombok.Getter;
 
 @Data
 public class Manifest {
   public String name;
   public String manifestVersion;
-  public List<String> jars;
   public Map<String, Object> options;
+  public Map<String, List<String>> resources;
```

Comment:
this change is related to updating the manifest.yml to handle deploying resources to all services. That's done by updating the plugin's manifest.yml to include a resources section instead of listing jars that go on java services. This is not a super necessary change, we could instead add a key to the manifest.yml that includes `deck` as a key and its values would be js files for Deck to consume